docker/tool/push-docker.sh

```diff
@@ -1,59 +1,60 @@
-#!/bin/bash
-
-# stop on error
-set -e
-
-# make sure we're in docker folder
-original_folder=$PWD
-
-if [ -z $"echo $PWD | grep /docker$" ]; then 
-    in_docker_folder=1
-else
-    in_docker_folder=0
-    cd docker
-fi
-
-function show_help
-{
-    echo "Usage:"
-    echo "push-docker.sh <dockerhub> <platform> [<version>]"
-    echo ""
-    echo "dockerhub: docker hub name (eg:microsoft/iotedgedev) used for pushing created images"    
-    echo "platform: linux|windows|empty"
-    echo "version: version to push. Automatically detected if not specified"
-    exit 1
-}
-
-IMAGE_NAME="$1"
-PLATFORM="$2"
-VERSION="$3"
-
-if [ "$IMAGE_NAME" = "--help" ]; then    
-    echo "Usage:"
-    echo "push-docker.sh imagename [linux|windows]"
-    exit 1
-fi
-
-if [ -z "$VERSION" ]; then
-    echo -e "\n===== Detecting version"
-    VERSION=$(cat ../iotedgedev/__init__.py | grep '__version__' | grep -oP "'\K[^']+")
-    echo "Detected version $VERSION"
-fi
-
-docker push $IMAGE_NAME:$VERSION-amd64 
-docker push $IMAGE_NAME:latest-amd64 
-
-#TODO add windows container support.  For now we only push linux.
-#docker push $IMAGE_NAME:$VERSION-windows-amd64
-#docker push $IMAGE_NAME:latest-windows-amd64
-
-echo -e "\n===== Creating Multi-Arch Docker image"
-#docker manifest create $IMAGE_NAME:latest $IMAGE_NAME:latest-amd64 $IMAGE_NAME:latest-windows-amd64 
-docker manifest create --insecure $IMAGE_NAME:latest $IMAGE_NAME:latest-amd64 
-
-echo -e "\n===== Pushing Docker Multi-Arch image"
-docker manifest push --purge $IMAGE_NAME:latest
-
-if [ in_docker_folder = 0 ]; then
-    cd original_folder
+#!/bin/bash

```

Comment:
did we change line endings or something? was it intentional?

Reply:
there is no diff in this line, will connect offline to udnerstand your question.

Reply:
on master:
![image](https://user-images.githubusercontent.com/11507404/121569119-4eab5380-c9d5-11eb-905a-f18c3c6bcb71.png)
on your branch:
![image](https://user-images.githubusercontent.com/11507404/121569202-62ef5080-c9d5-11eb-8942-a863bbe2f8ee.png)

Reply:
as you can see, on branch user/marine/mcrpush, the file is now using windows style line endings, which not good in a bash script :-)

Reply:
good catch, i will update